src/hotspot/share/opto/callnode.cpp

```diff
@@ -489,6 +489,7 @@ void JVMState::format(PhaseRegAlloc *regalloc, const Node *n, outputStream* st)
         ciField* cifield;
         if (iklass != NULL) {
           st->print(" [");
+          iklass->nof_nonstatic_fields();
```

Comment:
It's a workaround that initializes `ciInstanceKlass::_nonstatic_fields`.
Please, add a FIXME comment saying why you need it, so it will be fixed properly later.

Reply:
done